0085-Hex-Coverage-Limit.md

```diff
@@ -0,0 +1,89 @@
+
+| Radio |Signal Strength| Heartbeat Streak Start Date | Rank  | Coverage Points Per HIP 74| New Coverage Points|  
+|-------|---------------|-----------------------------|-------|---------------------------|--------------------|
+|   A   |   -77.33 dBm  |05/01/2023 23:24:25          | 1     | 16 (16 * 1)               | 16  (16 * 1)       |
+|   B   |   -88.75 dBm  |12/01/2022 01:01:01          | 2     | 16 (16 * 1)               | 12  (16 * .75)     |
+|   C   |   -88.75 dBm  |12/02/2022 12:11:01          | Fail  | 16 (16 * 1)               | 0   (16 * .5)      |
+|   D   |   -93.60 dBm  |12/05/2022 11:51:01          | Fail  | 16 (16 * 1)               | 0   (16 * 0)       |
+|   E   |   -94.69 dBm  |08/01/2022 05:01:59          | Fail  | 16 (16 * 1)               | 0   (16 * 0)       |
+
+Table Key:
+Radio:
+Signal Strength:
+Heartbeat:
+NEW Rank*:
+Coverage Points Per HIP 74:
+NEW Coverage Points:
+
+Table Explanation:
+
+Since Radio A has the highest signal strength in that hex, it will be ranked as "1", and granted a 1X multiplier, which will award it with the full 16 (16 x 1) modeled coverage points for that epoch.
+
+Since radios B and C tied in Signal Strength, the Heartbeat Streak date is used to determine which radio is ranked next. For this example, radio B had its Heartbeat Streak start on 12/01/2022 01:01:01 while radio C had a its HeartBeat streak start on 12/02/2022 12:11:01. Therefore, since radio B has an earlier Heartbeat Streak, it will be ranked "2", while radio C having a rank of "Fail".
+
+Since radio C, D and E had the lowest signal strength out of all five (5) radios, and only the top two (2) radios will earn PoC rewards, radios C, D and E will not earn any modeled coverage points, and are ranked as "Fail".
+
+< add here some info about the implications of this >
+
+## Drawbacks:
+The implementation of this proposal could increase the complexity of the Mobile network, and modeled coverage scores. Increasing complexity means...
+
+Additionally, radio deployers may lose out on awarded coverage points in instances where multiple radios are set up in the same hex.
+
+## Rationale and Alternatives:
+An alternative would be to allow radios and hexes to keep earning the defined amount of modeled coverage points as described in HIP 74. This may prevent or stagnate the growth of the network because this method does encourage the strategic placement of radios to minimize overlapping coverage. 
+
+## Unresolved Question:
+None
+
+## Deployment Impact:
+Outdoor radio deployers will now need to be cognizant of where they are placing their radios in order to maximize modeled coverage point. Additionally, there’s a large amount of overlapping coverage. Deployers may have to find new locations for some or all of their radios in order for them to continue to earn modeled coverage points. 
+
```

Comment:
What's the deployment impact on Helium Core Developers?
Speak to the Devs here. If they're doing this work, what should they know?